apps/dashboard/src/components/primitives/control-input/control-input.tsx

```diff
@@ -17,12 +18,13 @@ type CompletionRange = {
 };
 
 type ControlInputProps = {
+  className?: string;
   value: string;
   onChange: (value: string) => void;
   variables: LiquidVariable[];
   placeholder?: string;
   autoFocus?: boolean;
-  size?: 'default' | 'lg';
+  size?: 'base' | 'small';
```

Comment:
```suggestion
  size?: 'md' | 'sm';
```

We use the shorter md | sm | xs notation in all the other components and just put a default on MD or sm for the default state. So, for consistency's sake, I would suggest keeping it as the shorter notation.